genesyscloud/telephony_providers_edges_phonebasesettings/resource_genesyscloud_telephony_providers_edges_phonebasesettings_utils.go

```diff
@@ -130,3 +130,52 @@ func customizePhoneBaseSettingsPropertiesDiff(ctx context.Context, diff *schema.
 
 	return util.ApplyPropertyDefaults(diff, phoneBaseSetting.Properties)
 }
+
+func BuildTelephonyLineBaseProperties(d *schema.ResourceData) *map[string]interface{} {
+
+	if lineBase := d.Get("line_base").([]interface{}); len(lineBase) > 0 {
+
+		lineBaseMap := lineBase[0].(map[string]interface{})
```

Comment:
Since there are no required fields in the `line_base` schema, we should be careful when casting the item to a map as it might be nil.

```go
lineBaseMap := lineBase[0].(map[string]interface{})
``` 

⬇️ 

```go
lineBaseMap, ok := lineBase[0].(map[string]interface{})
if !ok {
  return nil
}
```

I guess we could either make this change or make `station_persistent_enabled` Required.